src/utils/hact/workOrderStatusUpdate/updateWorkOrder.js

```diff
@@ -32,7 +33,7 @@ export const buildWorkOrderUpdate = (
       variationReason.length && {
         comments: variationReason,
       }),
-    ...(isOvertime && { isOvertime: isOvertime }),
+    ...(isOvertimeUpdate && { isOvertime: isOvertime }),
```

Comment:
I know it was added in a previous PR but we should add a unit test for this new logic please.